src/Features/Core/Portable/OrganizeImports/IOrganizeImportsService.cs

```diff
@@ -10,6 +10,8 @@ internal interface IOrganizeImportsService : ILanguageService
     {
         Task<Document> OrganizeImportsAsync(Document document, CancellationToken cancellationToken);
 
+        string SortImportsDisplayStringWithAccelerator { get; }
```

Comment:
Is this an InternalsVisibleTo break that we need to be worried about if F# or TypeScript implement this?

Reply:
It would be. After searching their repos it doesn't appear that they use this interface.